styled.css

```diff
@@ -0,0 +1,307 @@
+@font-face {
```

Comment:
remove fonts/AvenirLTPro all fonts file as you are using only nexa here @PRABgit01

Reply:
ok ill fix it